docs/source/examples/plots_examples.rst

```diff
@@ -1,6 +1,169 @@
 Plotting with manim
 =================================
 
+Examples to illustrate the use of GrapheScenes in Manim
+
+
+.. manim:: Plot1
+    :quality: medium
+    :save_last_frame:
+
+    class Plot1(GraphScene):
+        def construct(self):
+            self.setup_axes()
+            my_func = lambda x: np.sin(x)
+            func_graph=self.get_graph(my_func)
+            self.add(func_graph)
+
+.. manim:: Plot2yLabel
+    :quality: medium
+    :save_last_frame:
+
+    class Plot2yLabel(GraphScene):
+        CONFIG = {
+            "y_min": 0,
+            "y_max": 100,
+            "y_tick_frequency": 10,
+            "y_labeled_nums": np.arange(0, 100, 10)
+        }
+
+        def construct(self):
+            self.setup_axes(animate=True)
+            dot = Dot()
+            dot.move_to(self.coords_to_point(PI / 2, 20))
+            my_func = lambda x: 20 * np.sin(x)
+            func_graph = self.get_graph(my_func)
+            self.add(func_graph)
+            self.add(dot)
+
+.. manim:: Plot3DataPoints
+    :quality: medium
+    :save_last_frame:
+
+    class Plot3DataPoints(GraphScene):
+        CONFIG = {
+            "y_axis_label": r"Concentration[$\%$]",
+            "x_axis_label": "Time [s]",
+            }
+
+        def construct(self):
+            data = [1, 2, 2, 4, 4, 1, 3]
+            self.setup_axes(animate=True)
+            for time, dat in enumerate(data):
+                dot = Dot().move_to(self.coords_to_point(time, dat))
+                self.add(dot)
+                self.wait(1)
+                print(time)
```

Comment:
This won't make any difference at all in the shown example so I'd suggest we remove it.

Reply:
done